### ISSUE TITLE
How large your arclose is when reach 99%+ accuracy.

### ISSUE THREAD

**Issue by JingyuanHu** [auroua/InsightFace_TF#52]
I try to reimplement the result by pytorch. However, I just get accuracy 96.7% on lfw when Arcloss is around 30. So I want to know how large arclose is when you reach 99%+ accuracy.

**Comment by ltcs11** [comment]
lfw acc 99.3% at training loss equals to 8
and lfw acc 99.0% at training loss equals to 12 
**just for reference**

**Comment by JingyuanHu** [comment]
@ltcs11 Thank you, it seems i have a lot of work to do, cause my loss is still much larger than 8 or 12. I'll try to increase my learning rate and betchsize.
